app/managers.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        os.chdir(r"C:\projects\projects\py-actor-manager")
```

Comment:
Avoid doing this operation since it is platform-specific, use only `sqlite3.connect`